tests/src/JIT/Directed/Misc/gettype/gettypetypeofmatrix.csproj

```diff
@@ -22,8 +22,7 @@
     </CodeAnalysisDependentAssemblyPaths>
   </ItemGroup>
   <PropertyGroup>
-    <!-- Set to 'Full' if the Debug? column is marked in the spreadsheet. Leave blank otherwise. -->
-    <DebugType></DebugType>
+    <DebugType>None</DebugType>
```

Comment:
@AndyAyersMS mentioned in another PR that this was functionally equivalent.  Is there a reason for modifying all these?